ci/source/op_opal_fvt.py

```diff
@@ -459,3 +458,29 @@ def test_list_pci_device_info():
         returns: int 0-success, raises exception-error
     """
     return opTestPCI.test_host_pci_devices_info()
+
+
+import os
+import unittest
+
+import ConfigParser
+from common.OpTestSystem import OpTestSystem
+from common.OpTestError import OpTestError
+from common.OpTestConstants import OpTestConstants as BMC_CONST
+import ci.source.op_inbound_hpm as op_inbound_hpm
+import ci.source.op_occ_fvt as op_occ_fvt
+import ci.source.op_fwts_fvt as op_fwts_fvt
+import ci.source.op_outofband_firmware_update as op_outofband_firmware_update
+import ci.source.op_firmware_component_update as op_firmware_component_update
+import ci.source.op_bmc_web_update as op_bmc_web_update
+
+class OpalPCI(unittest.TestCase):
```

Comment:
at this point I'm not quite sure, so I'm leaving the options open, maintaining the existing code as-is and just adding these small wrappers. I suspect that once things have been converted over everywhere we can go back and refactor things out to be in neat units.